tap_nice_incontact/schemas/qm_workflows.json

```diff
@@ -0,0 +1,133 @@
+{
+  "type": ["null", "object"],
+  "additionalProperties": false,
+  "properties": {
+    "tenantName": {
+      "type": ["null", "string"]
+    },
+    "agentId": {
+      "type": ["null", "string"]
+    },
+    "type": {
+      "type": ["null", "string"]
+    },
+    "evaluationFlowTrigger": {
+      "type": ["null", "string"]
+    },
+    "workflowStatus": {
+      "type": ["null", "string"]
+    },
+    "autoAcknowledged": {
+      "type": ["null", "string"]
+    },
+    "workflowState": {
+      "type": ["null", "string"]
+    },
+    "workflowId": {
+      "type": ["null", "string"]
+    },
+    "planName": {
+      "type": ["null", "string"]
+    },
+    "formName": {
+      "type": ["null", "string"]
+    },
+    "formVersion": {
+      "type": ["null", "string"]
+    },
+    "startDate": {
+      "type": ["null", "string"],
+      "format": "date-time"
+    },
+    "submissionDate": {
+      "type": ["null", "string"],
+      "format": "date-time"
+    },
+    "lastUpdated": {
+      "type": ["null", "string"],
+      "format": "date-time"
+    },
+    "scorePercentage": {
+      "type": ["null", "string"]
+    },
+    "score": {
+      "type": ["null", "string"]
+    },
+    "maximumPossibleScore": {
+      "type": ["null", "string"]
+    },
```

Comment:
are we also sure that these are strings?

Reply:
Updated to numbers